normandy/base/tests/api/test_authentication.py

```diff
@@ -241,3 +242,39 @@ def test_existing_user_with_no_email(self, csrf_api_client, mock_oidc):
 
         user.refresh_from_db()
         assert user.email == "john.doe@example.com"
+
+
+@pytest.mark.django_db
+@pytest.mark.urls("normandy.base.tests.api.test_authentication")
+class TestInsecureEmailAuthentication(object):
+    """Bearer token authentication"""
+
+    @pytest.fixture(autouse=True)
+    def clear_cache(self):
+        cache.clear()
+
+    @pytest.fixture
+    def bare_api_client(self):
+        return APIClient()
+
+    def test_it_works(self, bare_api_client):
```

Comment:
These tests work because of the default in `settings.py`, right? How are we testing the secure authentication then? Relying on the defaults in a far away file makes these tests harder to read.

Can you put the settings manipulation closer to this test, perhaps as a fixture, or figure out how the other authentication class does its test and mimic that?

Reply:
https://github.com/mozilla/normandy/pull/1839/files#diff-a63ebe522a13bf7f3c4e913c0268c906R37

Reply:
It's not relying on the default in `settings.py`.

Reply:
Ohhh. I didn't read closely enough. Nice!